bundles/CoreBundle/src/EventListener/Frontend/ElementListener.php

```diff
@@ -97,22 +95,6 @@ public function onKernelController(ControllerEvent $event)
                     $document->getFullPath(),
```

Comment:
What about this line above? Can't the !$request->attributes->get(self::FORCE_ALLOW_PROCESSING_UNPUBLISHED_ELEMENTS) be removed as well?
```php
   if ($document && !$document->isPublished() && !$user && !$request->attributes->get(self::FORCE_ALLOW_PROCESSING_UNPUBLISHED_ELEMENTS)) 
```